config/chainbridge-runtime-config.cere.dev.js

```diff
@@ -5,30 +5,30 @@ const cereTokenDecimals = 10;
 window.__RUNTIME_CONFIG__ = {
   CHAINBRIDGE: {
     chains: [
-      // ToDo: Uncomment when Polygon will be added
-      // {
-      //   chainId: 81,
-      //   networkId: 80001,
-      //   name: "Polygon Mumbai",
-      //   decimals: cereTokenDecimals,
-      //   bridgeAddress: "0x80C8ee1C89c574eB97B3783f0B9C9483EdF8B92E",
-      //   erc20HandlerAddress: "0x53cc2228587EAca7CcE3189f24BC9bd490E43C48",
-      //   rpcUrl: "",
-      //   type: "Ethereum",
-      //   nativeTokenSymbol: "MATIC",
-      //   tokens: [
-      //     {
-      //       address: "0xcD54b7066839B25A0E5fDf76866D338cc7257179",
-      //       name: cereTokenName,
-      //       symbol: cereTokenSymbol,
-      //       imageUri: "CEREIcon",
-      //       resourceId:
-      //         "0x000000000000000000000000000000c76ebe4a02bbc34786d860b355f5a5ce00",
-      //       decimals: cereTokenDecimals,
-      //       isDoubleApproval: false,
-      //     },
-      //   ],
-      // },
+      {
+        chainId: 81,
+        networkId: 80001,
+        name: "Polygon Mumbai",
+        decimals: cereTokenDecimals,
+        bridgeAddress: "0x80C8ee1C89c574eB97B3783f0B9C9483EdF8B92E",
+        erc20HandlerAddress: "0x53cc2228587EAca7CcE3189f24BC9bd490E43C48",
+        rpcUrl:
+          "https://polygon-mumbai.infura.io/v3/178b87e860404572bc217ca53fe594bd",
+        type: "Ethereum",
+        nativeTokenSymbol: "MATIC",
+        tokens: [
+          {
+            address: "0xcD54b7066839B25A0E5fDf76866D338cc7257179",
+            name: "CN",
+            symbol: "CS",
```

Comment:
cereTokenSymbol?

Reply:
While deploying the token symbol is set as 'CS'

Reply:
Ok, but let's keep it as on PROD for UI.